eng/pipelines/mgmt-ci.yml

```diff
@@ -4,6 +4,7 @@ trigger:
       - master
   paths:
     exclude:
+      - eng/common/
```

Comment:
I'm Ok with this for now but there is definitely a chance that mgmt depends on stuff from eng/common in the future and at that point we will need to reconcile.